src/Synapse/Synapse/help/Resume-AzSynapseSqlPool.md

```diff
@@ -156,6 +156,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Version
+Version of Synapse SQL pool. For example, 2 or 3.
+
+```yaml
+Type: System.Int32
```

Comment:
Normally version is a string, because it can contain "." or "-" or "beta".
But I guess the service models it as int so I'm not against the idea

Reply:
Thanks for the suggestion. This is very helpful. We are going to change the version to strings. I will discuss with PM side to finalize the concrete value of version strings to use.

Reply:
@isra-fel Talked to PM owner, we will keep the integer value.